src/main/java/frc/robot/subsystems/elbow/factory/RealElbowConstants.java

```diff
@@ -76,4 +77,25 @@ protected static ElbowStuff generateElbowStuff(String logPath) {
 		);
 	}
 
+	protected static Joint generateJointElbow(String logPath){
+		SparkMaxWrapper motor = new SparkMaxWrapper(IDs.CANSparkMAXs.ELBOW);
+		configMotor(motor);
+
+		SuppliedAngleSignal positionSignal = new SuppliedAngleSignal("position", () -> motor.getEncoder().getPosition(), AngleUnit.ROTATIONS);
+		SuppliedAngleSignal velocitySignal = new SuppliedAngleSignal("velocity", () -> motor.getEncoder().getVelocity(), AngleUnit.ROTATIONS);
+		SuppliedDoubleSignal currentSignal = new SuppliedDoubleSignal("output current", motor::getOutputCurrent);
+		SuppliedDoubleSignal voltageSignal = new SuppliedDoubleSignal("voltage", motor::getVoltage);
+
+		BrushlessSparkMAXMotor elbow = new BrushlessSparkMAXMotor(logPath, motor, new SysIdRoutine.Config());
```

Comment:
```suggestion
		BrushlessSparkMAXMotor motor = new BrushlessSparkMAXMotor(logPath, motor, new SysIdRoutine.Config());
```

Reply:
i want here to look at the constructor of joint and see what u give him (like u give him motor/signal/request not elbow)